packages/mobile/src/web3/saga.ts

```diff
@@ -218,9 +222,10 @@ async function savePrivateKeyToLocalDisk(
 ) {
   ensureAddressAndKeyMatch(account, privateKey)
   const filePath = getPrivateKeyFilePath(account)
-  Logger.debug('savePrivateKeyToLocalDisk', `Writing private key to ${filePath}`)
-  // TODO(ashishb): Store encrypted private key instead
-  await RNFS.writeFile(getPrivateKeyFilePath(account), privateKey)
+  const plainTextData = privateKey
```

Comment:
nit: why not skip the variable assignment and just pass privateKey into the function

Reply:
`getEncryptedData(privateKey, encryptionPassword)` felt a bit confusing to read.